### PR TITLE
Feat: display top 5 mews on home page

### PR DESCRIPTION
**Pull request by mattyg**
Display the top 5 mews of the day / week / month / year on the homepage, with multiple lists sorted by licks, mewmews, replies and quotes.

**Todo**
- [ ] ~Refactor to only have a subset of agents generate the top 5 list and save to an index, while others simply query the index~ (moving out to #47)
- [x] add support for linking to Records in holochain-time-index, switch dependency back to upstream holochain-time-index

### CODE REVIEW

**Comment by bierlingm** [comment]
Love it!

**Review by jost-s — APPROVED** [review]
Amazing work, Matty! I like this solution a lot.

There's the suggestion of a loading indicator that I would like to get in. But I'm also happy to merge this and do the loader in a follow up branch; your call.

File: ui/src/pages/HomePage.vue [hunk 1]
```diff
@@ -1,11 +1,40 @@
 <template>
   <q-page class="text-center" :style-fn="pageHeightCorrection">
-    <h6>Meeow, you're in!</h6>
-    <h2>Welcome to the Clutter</h2>
-    <q-img width="40%" src="@/assets/img/cat-eating-bird-circle.png" />
+    <div v-if="hasTopMewsLists">
```
[the remark this answers is not in the document]
Comment (mattyg) [reply ui/src/pages/HomePage.vue]: :facepalm:

File: ui/src/stores/clutter.ts [hunk 2]
```diff
@@ -16,35 +22,134 @@ export const MEWS_ZOME_NAME = "mews";
...
+          this.isLoadingTop[interaction] = true;
+          const [mews_day, mews_week, mews_month, mews_year] =
+            await Promise.all([
+              mewsTopList(interaction, {
+                count: 5,
+                from_seconds_ago: 24 * 60 * 60,
+              }),
+              mewsTopList(interaction, {
+                count: 5,
+                from_seconds_ago: 24 * 60 * 60 * 7,
```
[the remark this answers is not in the document]
Comment (mattyg) [reply ui/src/stores/clutter.ts:89]: :+1:

**Comment by jost-s** [comment]
@mattyg The time index DNA requires a bump to HDK 0.1, that's why this doesn't work yet.

**Comment by mattyg** [comment]
> @mattyg The time index DNA requires a bump to HDK 0.1, that's why this doesn't work yet.

Just a heads up when I talked with Art he wasn't thrilled about merging this feature, given that it doesn't scale (at least with this architecture). 

I think for our 0.1 release it makes more sense to implement the much simpler '#HomePage' feature noted in the hackmd specs (where every mew with the hashtag #HomePage is displayed on the home page).

Though I'll still try to upgrade the holochain-time-index library --- we'll need it in the future anyway for pagination / polling.

**Comment by mattyg** [comment]
Since we no longer want to implement this feature, I'm going to close this PR.

**Pull request closed without merging**